.github/workflows/gradle.yml

```diff
@@ -37,5 +35,43 @@ jobs:
       run: sudo apt-get install omlib-modelica-3.0
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
-    - name: Test with Gradle
-      run: ./gradlew test -i
+    - name: Build with Gradle
+      run: ./gradlew build -i
+  release:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out ${{ github.ref }}
+      uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
```

Comment:
I think I've said this before, but this should not be necessary if the file is checked into git with the correct permissions. Git is aware of file permissions.